interp/tests/complex/unsigned-dot-product.expect

```diff
@@ -71,7 +71,7 @@
         "addr1": 0,
         "clk": 0,
         "done": 0,
-        "read_data": 12,
+        "read_data": 0,
```

Comment:
Shouldn't this have value from the last addr passed with the read_en signal?

Reply:
This is an old test with `std_mem` not `seq_mem`